project_stock/views/project_project_view.xml

```diff
@@ -6,11 +6,7 @@
         <field name="inherit_id" ref="project.edit_project" />
         <field name="arch" type="xml">
             <xpath expr="///group[@name='misc']" position="inside">
-                <group
-                    string="Stock"
-                    name="stock_misc"
-                    groups="stock.group_stock_multi_locations"
-                >
+                <group string="Stock" name="stock_misc" groups="stock.group_stock_user">
```

Comment:
I think you should put both groups, here this one is correct, but the fields shouldn't be shown if no multi locations are checked.